create_db.sql

```diff
@@ -1,73 +1,88 @@
--- SQL script for creating the Database
+create database BarberShop
+use BarberShop
 
--- Create Departments and GroupCurators tables
-create database University;
-go
-use University;
-go
-
-create table Subjects 
```

Comment:
You should write this task by creating a new empty file in order to avoid code lines like these selected.